src/trans/ColorMap.ts

```diff
@@ -15,13 +16,21 @@ import RenderingContext from "../webgl/RenderingContext";
 import ShaderProgram from "../webgl/ShaderProgram";
 
 /**
- * Color Map Item for [[IColorMapOpts]]
+ * Color Map Item for [[IColorMapMap]]
  */
 export interface IColorMapItem {
-    index: number;
+    position: number;
     color: string;
 }
 
+/**
+ * A single Color Map for [[IcolorMapOpts]]
+ */
+export interface IColorMapMap {
```

Comment:
How about `IColorMapDef` or `IColorMapRow`? `IColorMapMap` sounds a bit misleading.

Reply:
Yeah, I agree, it's less than ideal. I'm leaning towards `IColorMapDef` or maybe `IColorMapEntry`?

Reply:
Let's go with `IColorMapDef`. I think having an `Entry` and an `Item` might be confusing.